src/lib/icons/index.tsx

```diff
@@ -133,3 +133,48 @@ export const Spinner = styled(icon(SpinnerIcon))<{ color?: Color }>`
   stroke-linecap: round;
   stroke-width: 2;
 `
+
```

Comment:
This is long enough that I think it should go into its own file, src/lib/icons/DetailedSpinner.
Also, everything exported from this file is an Icon. Do the types work in Caption, or does the definition of Icon need to be expanded?

Reply:
types actually work fine in Caption no changes needed, but yeah can move this into its own file.